source/core.py

```diff
@@ -30,21 +30,30 @@
 ALL_GOVERNORS = ("performance", "ondemand", "conservative", "schedutil", "userspace", "powersave")
 CPUS = os.cpu_count()
 
-# Note: 
+# Note:
 # "load1m" & "cpuload" can't be global vars and to in order to show correct data must be
 # decraled where their execution takes place
 
 # powersave/performance system load thresholds
 powersave_load_threshold = (75*CPUS)/100
 performance_load_threshold = (50*CPUS)/100
 
-# auto-cpufreq log file
```

Comment:
I like where you're going with this. But I'm not sure if it's due to these changes or some other changes. But if I install auto-cpufreq using `auto-cpufreq-installer --install` and then run `auto-cpufreq --install` I'll end up with following error:

```
FileNotFoundError: [Errno 2] No such file or directory: '/var/log/auto-cpufreq.log'
```

Reply:
Should be fixed now, It now only opens the log file if it is needed.

Reply:
Yes, thanks it's fixed but now the log is empty as reported in [comment above](https://github.com/AdnanHodzic/auto-cpufreq/pull/148#discussion_r561645729).